themes/nekonight/nekonight.theme.sh

```diff
@@ -1,55 +1,23 @@
 # nekonight Bash prompt with source control management
 # Author: Bruno Ciccarino <brunociccarinoo@gmail.com>
 #
-#
 # Theme inspired by:
 #  - Bash_it cupcake theme
 # Demo:
 # ╭─🐱 virtualenv 🐱user at 🐱host in 🐱directory on (🐱branch {1} ↑1 ↓1 +1 •1 ⌀1 ✗)
 # ╰λ cd ~/path/to/your-directory
 
+source "$OSH/themes/nekonight/nekonight-base.sh"
+
+_omb_theme_nekonight_icon_emoji="🐱"
```

Comment:
The variable `_omb_theme_nekonight_icon_emoji` is :+1:, but then I feel the other variables `icon_*` should also use this variable `_omb_theme_nekonight_icon_emoji`.